io.github.igorlogius.scr2ppm.yml

```
@@ -0,0 +1,25 @@
+app-id: io.github.igorlogius.scr2ppm
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: scr2ppm
+rename-icon: scr2ppm
+rename-desktop-file: scr2ppm.desktop
+rename-appdata-file: scr2ppm.metainfo.xml
+finish-args:
+  - --socket=x11
+  - --socket=ipc
```

```suggestion
  - --share=ipc
```

I wrote it fine the first time.